config/schemas/settings.json

```diff
@@ -101,6 +104,18 @@
 				"longTouchTimeMilliseconds" : {
 					"type" : "number",
 					"default" : 1000
+				},
+				"autosaveCountLimit" : {
+					"type" : "number",
+					"default": 5
+				},
+				"useSavePrefix" : {
+					"type": "boolean",
+					"default": false
+				},
+				"savePrefix" : {
+					"type": "string",
+					"default": ""
```

Comment:
Not sure if there is any need of fixed save prefix. After all, this is prefix for *all* saves, see 0 possible use-cases here.

Proposition: remove